ospf6d/ospf6_top.h

```diff
@@ -38,6 +38,23 @@ enum {
 	OSPF6_LOG_ADJACENCY_DETAIL =	(1 << 1),
 };
 
+struct ospf6_redist {
+	unsigned short instance;
+	/* Redistribute metric info. */
+	struct {
+		int type;  /* External metric type (E1 or E2).  */
```

Comment:
I see places in the code where type is a uint8_t, places it is a uint16_t.  Let's choose one and stick with it.

Reply:
made 
uint8_t type; 
uint32_t value;